satpy/dependency_tree.py

```diff
@@ -343,13 +339,13 @@ def _get_unique_id_from_sorted_ids(sorted_ids, distances):
             raise MissingDependencies
         return result
 
-    def _get_unique_node_from_id(self, result, data):
+    def _get_unique_reader_node_from_id(self, unique_id, reader_name):
```

Comment:
`unique_id`? Why not call it `data_id` if that's what it is?

Reply:
Good point, thanks